evm-contracts/test/v0.6/BlockhashStore.test.ts

```diff
@@ -0,0 +1,132 @@
+      hash: "0x325fde74e261fc483a16506bbc711b645b043ad24c7db0136845a1be262cf0c9",
+    },
+    {
+      num: 10000470,
+      rlpHeader: ethers.utils.arrayify("0xf90215a0325fde74e261fc483a16506bbc711b645b043ad24c7db0136845a1be262cf0c9a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794ea674fdde714fd979de3edf0f56aa9716b898ec8a020647cfa35563093442a12d80bf2bacb83da1de8340366677f3822591a334ccea066ad7285f6c5b6407f62c6b65a83aeaaa71ad9a97c2bb15139140f2dbb60f7e0a0c0e633851d0b5ce661ecc054517425e82425fcc6170db9693e5b5a6dd5ef6d6bb90100c0c000c1520708182080c8e461891c2402800a80d44a00034259414012012a5006a1416331181504902044960808f1129018800311621e920886804693749b10542400142e984580ccba634881c4156962200ecfb004000005468db44842781c59923110262660802315006106388b028412c42c000820c508e66b7851fa68002008144cd7860cd884280802915163399c168d5a11b0649486084110149469a1e61c31134204b903206566885180bc0426c0c6c0a4d408e182242f08180d204c624a040248425041ac028010d088820402ba4bd38c2d1215829300543465603822110500811290490148049300040e000c280086a09e8100089818ce480a887e87086c4965bf3c8a839898568398705c839847d2845eb02fe994505059452d65746865726d696e652d6575312d35a09d8ae288d0eede524f3ef5e6cfcc5ba07f380bc695bb71578a7b91cfa517071b8859d0976006378e52"),
+      hash: "0x5cf096dfd1fc2d2947a96fdec5377ab7beaa0eb00c80728a3b96f0864cec506a",
+    },
+  ]
+
+  const deployment = setup.snapshot(provider, async () => {
+    blockhashStoreTestHelper = await blockhashStoreTestHelperFactory.connect(personas.Default).deploy()
+  })
+
+  beforeEach(async () => {
+    await deployment()
+
+    const [lastBlock] = mainnetBlocks.slice(-1);
+    await blockhashStoreTestHelper.connect(personas.Default)
+      .godmodeSetHash(lastBlock.num, lastBlock.hash)
+    assert.strictEqual(await blockhashStoreTestHelper.getBlockhash(lastBlock.num), lastBlock.hash)
+  })
+
+  it('getBlockhash reverts for unknown blockhashes', async () => {
+    matchers.evmRevert(
```

Comment:
Do you need `await` here, too, as you've done for the other `matchers.evmRevert`s?